CONTRIBUTING.md

```diff
@@ -15,6 +15,24 @@ merging pull requests as well as a "lead maintainer" who is responsible for the
 release cycle, overall merging, moderation and appointment of maintainers.
 
 
+Communication Channels
+----------------------
+
+Most communication about Bitcoin Core development happens on IRC, in the
+#bitcoin-core-dev channel on Freenode. The easiest way to participate on IRC is
+with the web client, [webchat.freenode.net](https://webchat.freenode.net/). Chat
+history logs can be found
+on [botbot.me](https://botbot.me/freenode/bitcoin-core-dev/).
+
+Discussion about code base improvements happens in GitHub issues and on pull
+requests.
+
+The developer
+[mailing list](https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-dev)
```

Comment:
Maybe also list the bitcoin-core-dev mailing list: https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-core-dev although it isn't used except for release announcements. bitcoin-dev is more generally for Bitcoin things rather than Bitcoin Core things.

Reply:
If it's unused, I'd rather not list it. Unless we want to revive it.